install_locally.ps1

```diff
@@ -0,0 +1,87 @@
+<#
```

Comment:
Can you add in the docs a recommended use for this tool?

Reply:
Done